base/system/services/database.c

```diff
@@ -1449,15 +1552,15 @@ ScmControlService(HANDLE hControlPipe,
                 if (bResult == FALSE)
                 {
                     dwError = GetLastError();
-                    DPRINT1("GetOverlappedResult() failed (Error %lu)\n", dwError);
+                    DPRINT("(%S, %d) GetOverlappedResult() failed (Error %lu)\n", lpService->lpServiceName, dwControl, dwError);
```

Comment:
Hermes-style, I won't repeat for all of them, I guess you spotted the systematic already:
```suggestion
                    DPRINT("GetOverlappedResult(%S, %d) failed (Error %lu)\n", lpService->lpServiceName, dwControl, dwError);
```

Reply:
Should all be changed now. Thanks.

Reply:
@joachimhenze Yes this was my idea originally, but I noticed it would work only where I suggested it (because these were the actual parameters of the function), but not in all these other instances.